Mathlib/Data/Finset/Card.lean

```diff
@@ -130,13 +130,13 @@ theorem card_doubleton (h : a ≠ b) : ({a, b} : Finset α).card = 2 := by
   rw [card_insert_of_not_mem (not_mem_singleton.2 h), card_singleton]
 #align finset.card_doubleton Finset.card_doubleton
 
-/-- $\#(s \setminus \{a\}) = \#s - 1$ if $a \in s$. -/
+/-- $\\#(s \setminus \\{a\\}) = \\#s - 1$ if $a \in s$. -/
 @[simp]
 theorem card_erase_of_mem : a ∈ s → (s.erase a).card = s.card - 1 :=
   Multiset.card_erase_of_mem
 #align finset.card_erase_of_mem Finset.card_erase_of_mem
 
-/-- $\#(s \setminus \{a\}) = \#s - 1$ if $a \in s$.
+/-- $\\#(s \setminus \\{a\\}) = \\#s - 1$ if $a \in s$.
```

Comment:
As commented on Zulip, I think this is indicative of our markdown parser having issues, that we should fix there instead.